fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -58,7 +58,12 @@ def run
         c.description = 'Helps you with your initial fastlane setup'
 
         c.action do |args, options|
-          Fastlane::Setup.new.run
+          if args[0] == 'beta' && FastlaneCore::Feature.enabled?('FASTLANE_ENABLE_CRASHLYTICS_BETA_INITIALIZATION')
```

Comment:
Is the command that developers will use `init beta` or `beta_init`? I'm a little confused by this.